api/views.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```

Comment:
It was leftover from the bad old days before `from django.utils import timezone` and `timezone.now()` I suspect.